### ISSUE TITLE
Questions about TransformerNetModel initialization

### ISSUE THREAD

**Issue by QQQrookie** [Shark-NLP/DiffuSeq#11]
Hi:
Thank you for your excellent work and for providing a nice codebase.
First of all, I want to point out a bug. If I use the `bert-base-uncased` model to initialize, and word_embedding uses bert embedding, which dimension is 768, but `input_dims` is still 128, an error will be reported in the `forward()`.
Secondly, I would like to ask if all your experiments were not initialized with the pre-trained model. I see that `use_plm_init` defaults to "no" in the script you provided. I tried to use the bert model for initialization, but the loss seems abnormal. Have you tried this?

Any help is appreciated.

**Comment by summmeer** [comment]
@QQQrookie Hi,
we didn't use the `bert-base-uncased` model to initialize `word_embedding`. The dimensions 128 and 768 are projected by `self.input_up_proj`. We didn't initialize the model with the pre-trained model, just leaving the code implementation. You are free to try.

**Comment by QQQrookie** [comment]
@summmeer Thank you for the prompt reply.

**Comment by xiang-xiang-zhu** [comment]
> Hi: Thank you for your excellent work and for providing a nice codebase. First of all, I want to point out a bug. If I use the `bert-base-uncased` model to initialize, and word_embedding uses bert embedding, which dimension is 768, but `input_dims` is still 128, an error will be reported in the `forward()`. Secondly, I would like to ask if all your experiments were not initialized with the pre-trained model. I see that `use_plm_init` defaults to "no" in the script you provided. I tried to use the bert model for initialization, but the loss seems abnormal. Have you tried this?
> 
> Any help is appreciated.

Hello, how can I solve the first problem? And have you tried to use Bert?
